scripts/actions/report-deno-not-ran.ts

```diff
@@ -0,0 +1,87 @@
+import { Octokit } from '@octokit/action';
```

Comment:
If you really want TS then fine but IMHO you're adding a LOT of overhead (`ts-node`, extra tsconfig file, extra code in `tsconfig.eslint.json`, extra script in package.json, not having TLA) by having that over a `.mjs` file.

Personally I used to do TS based scripts too... 2 years ago. But ESM is just so much easier. Especially now that you've written (and presumably tested) it, you don't *really* need the type information anymore.

Reply:
I couldn't get ts-node to properly make the script it runs run as ESM (because node reads from package.json and well..yknow)

Having TLA is really not something I crave thaaat much, and the small extras which I can probably remove now. I'll wait for thoughts from the rest, but imo I'd say it's fine, especially since its fast nuff.